Runtime/CourseController/AdvancedCourseController.cs

```diff
@@ -520,15 +532,23 @@ private void PopulateChapterPickerOptions(int startingIndex)
 
                 // Reset the chapter picker.
                 chapterPicker.ClearOptions();
-
+                
                 // Populate it with new options.
                 chapterPicker.AddOptions(dropdownOptions);
-
+                
                 // Reset the selected value
                 chapterPicker.value = 0;
-
+                
                 // If there is only one option, the dropdown is currently disabled.
                 chapterPicker.enabled = dropdownOptions.Count > 1;
+                if (dropdownOptions.Count > 1 == false)
```

Comment:
```suggestion
                if (dropdownOptions.Count < 2)
```
Seems more readable to me. But I am also fine without the change.